card/mod/01_core/set/all/utils.rb

```diff
@@ -74,6 +74,11 @@ def merge name, attribs={}, opts={}
     end
   end
 
+
+  def older_than_five_days? time
+    Time.now - time > 432000
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.